### PR TITLE
First pass at overview

### PR DESCRIPTION
**Pull request by ggam**
@bshannon I left `overview009.adoc` untouched. My understanding is that Jakarta EE 8 has the same Java SE dependencies that Java EE 8.

### CODE REVIEW

File: src/main/jbake/content/overview008.adoc [hunk 1]
```diff
@@ -118,106 +117,106 @@ The following features support the GUI components:
...
+directly into a text-based document. A Jakarta page is a text-based document
 that contains two types of text:
 
 * Static data, which can be expressed in any text-based format, such as
 HTML or XML
 * JSP elements, which determine how the page constructs dynamic content
 
-For information about JSP technology, see _The Jakarta EE 5 Tutorial_ at
+For information about JSP technology, see _The Java EE 5 Tutorial_ at
 `http://docs.oracle.com/javaee/5/tutorial/doc/`.
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: It's weird that Jakarta Server Pages is referring to an old Java EE tutorial, and it's
unfortunate that all the JSP content was removed from the current version of the
tutorial.  I think we at least need a sentence something like this at the beginning
of this paragraph... "The Jakarta Server Pages technology is derived from and compatible
with the JavaServer Pages (JSP) technology."

File: src/main/jbake/content/overview008.adoc [hunk 2]
```diff
@@ -241,22 +240,22 @@ Managed Beans
...
 The Managed Beans specification is part of the Jakarta EE 8 platform
-specification (JSR 366). The Jakarta EE 8 platform requires Managed Beans
+specification. The Jakarta EE 8 platform requires Managed Beans
 1.0.
 
 [[GJXVO]][[contexts-and-dependency-injection-for-jakarta-ee]]
 
-Contexts and Dependency Injection for Jakarta EE
+Jakarta Contexts and Dependency Injection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: underline length

File: src/main/jbake/content/overview008.adoc [hunk 3]
```diff
@@ -273,17 +272,17 @@ The Jakarta EE 8 platform requires CDI 2.0.
 
 [[GJXVG]][[dependency-injection-for-java]]
 
-Dependency Injection for Java
+Jakarta Dependency Injection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: underline length

File: src/main/jbake/content/overview008.adoc [hunk 4]
```diff
@@ -330,62 +329,62 @@ resource manager of the EIS. Because a resource adapter is specific to
...
 
 [[BNACS]][[javamail-api]]
 
-JavaMail API
+Jakarta Mail
 ~~~~~~~~~~~~
 
-Jakarta EE applications use the JavaMail API to send email notifications.
-The JavaMail API has two parts:
+Jakarta EE applications use the Jakarta Mail to send email notifications.
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: Remove "the".

File: src/main/jbake/content/overview008.adoc [hunk 5]
```diff
@@ -330,62 +329,62 @@ resource manager of the EIS. Because a resource adapter is specific to
...
+The Jakarta Mail has two parts:
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: Remove "The".

File: src/main/jbake/content/overview008.adoc [hunk 6]
```diff
@@ -330,62 +329,62 @@ resource manager of the EIS. Because a resource adapter is specific to
...
 
 * An application-level interface used by the application components to
 send mail
 * A service provider interface
 
-The Jakarta EE platform includes the JavaMail API with a service provider
+The Jakarta EE platform includes the Jakarta Mail with a service provider
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: Remove "the".

File: src/main/jbake/content/overview008.adoc [hunk 7]
```diff
@@ -397,12 +396,12 @@ message sender. Authentication providers authenticate each incoming
...
-The Jakarta EE 8 platform requires JASPIC 1.1.
+The Jakarta EE 8 platform requires Jakarta Authentication 1.1.
 
 [[jakarta-ee-security-api]]
-Jakarta EE Security API
-~~~~~~~~~~~~~~~~~~~~~~~
-The Jakarta EE Security API specification defines portable, plug-in interfaces for
+Jakarta Security
+~~~~~~~~~~~~~~~~
+Jakarta Security specification defines portable, plug-in interfaces for
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: Insert "The".

File: src/main/jbake/content/overview008.adoc [hunk 8]
```diff
@@ -425,33 +424,33 @@ interact with the current security context. The specification also provides
...
 [[CJAHDJBJ]][[java-api-for-websocket]]
 
-Java API for WebSocket
-~~~~~~~~~~~~~~~~~~~~~~
+Jakarta WebSocket
+~~~~~~~~~~~~~~~~~
 
 WebSocket is an application protocol that provides full-duplex
-communications between two peers over TCP. The Java API for WebSocket
+communications between two peers over TCP. The Jakarta WebSocket
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: Remove "The".

File: src/main/jbake/content/overview008.adoc [hunk 9]
```diff
@@ -465,32 +464,32 @@ Defines a format and processing rules for applying operations to a JSON document
...
+The Jakarta EE 8 platform requires Jakarta JSON Processing 1.1.
 
 [[java-api-for-json-binding]]
-Java API for JSON Binding
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Jakarta JSON Binding
+~~~~~~~~~~~~~~~~~~~~
 
-The Java API for JSON Binding (JSON-B) provides a binding layer for converting Java objects to and from JSON messages. JSON-B also supports the ability to customize the default mapping process used in this binding layer through the use of Java annotations for a given field, JavaBean property, type or package, or by providing an implementation of a property naming strategy.
+The Jakarta JSON Binding provides a binding layer for converting Java objects to and from JSON messages. Jakarta JSON Binding also supports the ability to customize the default mapping process used in this binding layer through the use of Java annotations for a given field, JavaBean property, type or package, or by providing an implementation of a property naming strategy.
```
Comment (bshannon) [inline src/main/jbake/content/overview008.adoc]: Remove "The".

File: src/main/jbake/content/overview010.adoc [hunk 10]
```diff
@@ -61,17 +61,17 @@ client container libraries and JAR files.
 |Apache Derby |A copy of Apache Derby database.
 
 |`xjc` |A command-line tool to transform, or bind, a source XML schema
-to a set of JAXB content classes in the Java programming language.
+to a set of Jakarta XML Binding content classes in the Java programming language.
```
Comment (bshannon) [inline src/main/jbake/content/overview010.adoc]: This needs to stay "JAXB".

File: src/main/jbake/content/overview010.adoc [hunk 11]
```diff
@@ -61,17 +61,17 @@ client container libraries and JAR files.
...
 
 |`schemagen` |A command-line tool to create a schema file for each
 namespace referenced in your Java classes.
 
-|`wsimport` |A command-line tool to generate JAX-WS portable artifacts
+|`wsimport` |A command-line tool to generate Jakarta XML Web Services portable artifacts
```
Comment (bshannon) [inline src/main/jbake/content/overview010.adoc]: This needs to stay "JAX-WS".

File: src/main/jbake/content/overview010.adoc [hunk 12]
```diff
@@ -61,17 +61,17 @@ client container libraries and JAR files.
...
 for a given WSDL file. After generation, these artifacts can be packaged
 in a WAR file with the WSDL and schema documents, along with the
 endpoint implementation, and then deployed.
 
 |`wsgen` |A command-line tool to read a web service endpoint class and
-generate all the required JAX-WS portable artifacts for web service
+generate all the required Jakarta XML Web Services portable artifacts for web service
```
Comment (bshannon) [inline src/main/jbake/content/overview010.adoc]: Ditto.

File: src/main/jbake/content/overview008.adoc (hunk 1 above)
Comment (ggam) [reply src/main/jbake/content/overview008.adoc]: Great suggestion, thanks

**Pull request merged**